media/js/mozorg/home/takeover-2016.js

```diff
@@ -67,18 +70,21 @@
         $options.removeAttr('checked');
     }
 
-    // Shows the take over modal, sets the required entry in
-    // session storage and adds an `esc` key handler to close
-    // the modal.
+    // Shows the take over modal, sets the seven day cookie
+    // and adds an `esc` key handler to close the modal.
     function initTakeOver() {
         // format the currency in browsers that support it
         if (intlNumberFormatSupported) {
             formatCurrency();
         }
-        // show the takeover
-        toggleModal();
-        // set mozorg.takeover.seen to true
-        storeSession();
+
+        // Don't show the takeover if cookies are disabled or not supported
+        if (document.cookie) {
```

Comment:
Fwiw, this is the Modernizr feature detect for cookies if you wanted to add a helper function: https://github.com/Modernizr/Modernizr/blob/master/feature-detects/cookies.js